tests/e2e/specs/js-scripts/wp-consent-api.test.js

```diff
@@ -84,6 +84,66 @@ test.describe( 'WP Consent API Integration', () => {
 		} );
 	} );
 
+	test( 'Consent update denying `analytics_storage` is sent when WP Consent API `statistics` category is `denied`', async ( {
+		page,
+	} ) => {
+		await page.goto( 'shop?consent_default=granted' );
```

Comment:
Might the `consent_default` parameter be omitted? This would be a bit more consistent with this test case:

https://github.com/woocommerce/woocommerce-google-analytics-integration/blob/88f9ea30ac687eccc84248fbeb557081a61b0280/tests/e2e/specs/js-scripts/wp-consent-api.test.js#L184-L191

But if the purpose of specifying this parameter is to explicitly express its initial value, then I think that's also good, and perhaps the parameter could be added to the last test case as well.